lib/filesystem.js

```diff
@@ -138,7 +138,7 @@ exports.readChunk = (fd, size, position, callback, retries) => {
     if (error) {
       const platform = os.platform();
 
-      if (error.code === 'ENXIO' && platform === 'darwin') {
+      if (platform === 'darwin' && (error.code === 'ENXIO' || error.code === 'EBUSY')) {
```

Comment:
Haha, that's clever - done by rebasing on top of https://github.com/resin-io-modules/etcher-image-write/commit/3becd3d6fae7f6fe43690c50e2cb1ac998118cc0 :-)